10.9-libcxx/stable/main/finkinfo/sci/qhull8.0-shlibs.info

```diff
@@ -0,0 +1,112 @@
+Package: qhull8.0-shlibs
+Version: 2020.2
+Revision: 1
+Description: Calculate convex hulls and related structures
+License: BSD
+Maintainer: None <fink-devel@lists.sourceforge.net>
+DescDetail: <<
+Qhull computes convex hulls, Delaunay triangulations, halfspace 
+intersections about a point, Voronoi diagrams, furthest-site 
+Delaunay triangulations, and furthest-site Voronoi diagrams. 
+It runs in 2-d, 3-d, 4-d, and higher dimensions. It implements 
+the Quickhull algorithm for computing the convex hull. Qhull 
+handles roundoff errors from floating point arithmetic. It 
+computes volumes, surface areas, and approximations to the 
+convex hull.
+<<
+DescUsage: <<
+This package contains the shared libraries.
+<<
+DescPackaging: <<
+* Previous maintainer: Alexander Hansen
+* The prior static-only qhull package was maintained by Stefan Langerman 
+  <fink@slef.org>.
+* Upstream has decided to build the executables statically, so Depends:
+  %N is not required.
+* Patch library install_name to be a full and not relative path.
+<<
+BuildDepends: <<
+	cmake,
+	fink (>= 0.30.0),
+	fink-buildenv-modules
+<<
+
+Source: https://github.com/qhull/qhull/archive/%v.tar.gz
+SourceRename: qhull-%v.tar.gz
+Source-MD5: 123bf4764c374a645b34d44af7853799
+
+PatchFile: %n.patch
+PatchFile-MD5: d01385e1d7a5a16ac40551e7962cc0f6
+GCC: 4.0
+CompileScript: <<
+	#!/bin/sh -ev
+	. %p/sbin/fink-buildenv-cmake.sh
+	mkdir finkbuild
+	cd finkbuild
+	cmake $FINK_CMAKE_ARGS \
+		-DBIN_INSTALL_DIR=%p/bin \
+		-DDOC_INSTALL_DIR=%p/share/doc/qhull \
+		-DMAN_INSTALL_DIR=%p/share/man/man1 \
+		..
+	make -w
+<<
+
+InfoTest: <<
+	TestScript: <<
+		#!/bin/sh -ev
+		export PATH=$PATH:%b/bin
+		export DYLD_LIBRARY_PATH=%b/lib
+		cd finkbuild
+		ctest || exit 2
+	<<
+<<
+
+InstallScript: <<
+	#!/bin/sh -ev
+	cd finkbuild
+	make install DESTDIR=%d
+	#pushd %i/lib
+	# make sure that the unversioned dylibs are actually symlinks
+	#rm libqhull.dylib libqhull_p.dylib
+	#ln -s libqhull.6.3.1.dylib libqhull.dylib
+	#ln -s libqhull_p.6.3.1.dylib libqhull_p.dylib
+<<
+Shlibs: <<
+	%p/lib/libqhull_r.8.0.dylib 8.0.0 %n (>= 2020.2-1)
+<<
+
+DocFiles: Announce.txt COPYING.txt README.txt REGISTER.txt
+Homepage:  http://www.qhull.org/
+SplitOff:  <<
+	Package: qhull
+	Conflicts: qhull, qhull5, qhull6, qhull6.3, qhull6.3.1, qhull
+	Replaces: qhull, qhull5, qhull6, qhull6.3, qhull6.3.1, qhull
+	Files: <<
+		bin
+		share/doc/qhull
+		share/man
+	<<
+	DocFiles: Announce.txt COPYING.txt README.txt REGISTER.txt index.htm
+	DescUsage: <<
+	This package contains the executables, manpages, and other documentation.
```

Comment:
Might want to comment on the unusual situation of a binaries pkg not having dependencies on its own -shlibs (I guess it's static-linked?)

Reply:
In `DescPackaging`:
```
* Upstream has decided to build the executables statically, so Depends:
  %N is not required.
```
Moved the `qhull` package to remove ambiguity.